src/cluster_legacy.c

```diff
@@ -69,6 +69,7 @@ int clusterDelSlot(int slot);
 int clusterDelNodeSlots(clusterNode *node);
 int clusterNodeSetSlotBit(clusterNode *n, int slot);
 void clusterSetPrimary(clusterNode *n, int closeSlots);
+static inline int clusterNodeIsVotingPrimary(clusterNode *n);
```

Comment:
It is uncommon to separate the declaration and definition of a "static inline" function. Can you pull up the definition of "clusterNodeIsVotingPrimary" and remove this declaration?

Reply:
yes... it bother me as well, i now moved the definition to the up.